assets/app/components/nav.js

```diff
@@ -12,6 +12,8 @@ const propTypes = {
 const getSecondaryNav = (loggedIn, username) => {
   const authLinks = [
     <Link to="/sites">{username}</Link>,
+    <a href="https://federalist-docs.18f.gov" target="_blank">Documentation</a>,
+    <a href="https://github.com/18F/federalist/issues/new" target="_blank">Contact us</a>
```

Comment:
Looks like there's a syntax error from a missing comma at the end of this line.

Also, note that ES6 gives us [trailing commas](http://dontkry.com/posts/code/trailing-commas.html), so if you wanted you could also stick a comma to the end of line 17 below to reduce diff noise and make issues with commas more obvious in the future.